helm/charts/hpe-csi-driver/templates/cv-csp.yaml

```diff
@@ -0,0 +1,49 @@
+---
+### CSP deployment ###
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: csp-service
+  namespace: {{ .Release.Namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: cv-csp
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: cv-csp
+    spec: 
+      priorityClassName: system-cluster-critical
+      serviceAccount: hpe-csi-controller-sa
+      containers:
+        - name: cv-csp
+          {{- if .Values.registry }}
+          image: {{ .Values.registry }}/hpestorage/cv-csp:latest
```

Comment:
This should reflect the same version as the default below.

Reply:
done